teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(age:, specialization:)
```

Comment:
- [ ] Please add `name` and `parent permission` as parameters to the initialize method. It can look something similar to the code below
```
def initialize(age, specialization, name = "Unknown", parent_permission = true)
```

Reply:
@Lucas-Masaba Thank you for the feedback but from the instructions, the name and parent_permission  are optional parameters with default values in the `base class (Person)`, so I don't need to add them as a required parameter in the `subclasses(Student, Teacher)`